src/index.js

```diff
@@ -140,6 +140,19 @@ var VERSION = process.env.npm_package_version;
  * @param {boolean} [commit] The user action to show on the PayPal review page. If true, a `Pay Now` button will be shown. If false, a `Continue` button will be shown.
  */
 
+/** @typedef {object} applePayCreateOptions The configuration options for Apple Pay.
+ *
+ * @param {string} [buttonStyle=`black`] Configures the Apple Pay button style. Valid values are `black`, `white`, `white-outline`. Default value is `black`.
+ * @param {string} displayName The canonical name for your store. Use a non-localized name. This parameter should be a UTF-8 string that is a maximum of 128 characters. The system may display this name to the user.
+ * @param {external:ApplePayPaymentRequest} paymentRequest The payment request details to apply on top of those from Braintree.
```

Comment:
Should we mention what details are provided by Braintree here?

Reply:
I'm 👍 for that

Reply:
Think we're gonna beef up the jsdocs in a separate PR for easier reviewing